Gemfile

```diff
@@ -17,7 +17,7 @@ if defined?(JRUBY_VERSION)
 end
 
 # protocol parsers
-gem 'json', '>= 2.4.0' # due to load_file support
+gem 'json', '>= 2.4.0' # due to load_file support, see: https://github.com/ruby/json/issues/696
 gem 'nokogiri', '>= 1.6.8.1'
```

Comment:
Curious if anyone know of any context for this version min....

Reply:
The specific version (`= 1.6.8.1`) was added in 6a0dd35acab3ff8b1e4e971ce7c3d5d452ad8195 as the last version being compatible with Ruby 1.x. The `>=` was added in e62c1d553cc9c16b17b79866aad7f9fbb9ceeaf8.

Since the original argument for listing that specific version number no longer applies, I would say it’s ok to just remove it.

Reply:
Yep, I think you can remove it too. I recall some issue with nokogiri years ago where the supported ruby version requirement of the latest version did not work with the syntax of the shipped gem (newer syntax used in a gem that said it supported older version).